.github/workflows/reusable-build.yml

```diff
@@ -215,7 +215,19 @@ jobs:
       uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce
       with:
         name: ebpf-for-windows - NuGet package (${{inputs.build_artifact}}_${{env.BUILD_CONFIGURATION}})
-        path: ${{github.workspace}}/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}/*.nupkg
+        path: ${{github.workspace}}/${{env.BUILD_PLATFORM}}/${{env.BUILD_CONFIGURATION}}/eBPF-for-Windows.*.nupkg
+
+    - name: Build the NuGet Redist package
```

Comment:
Suggestion: Replace `Redist` by `Binary` ?

Reply:
@shankarseal I kept the naming consistent with the project and artifacts (and resonance with other redistributables like the VC++ Redist), no problem in changing it across all if not ok.